pfcpiface/up4_test.go

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Open Networking Foundation
+
+package pfcpiface
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_UP4_initTunnelPeerIDs(t *testing.T) {
+	type args struct {
+		mockUP4 *UP4
```

Comment:
This is not a mock. A mock replaces a dependency of the class under test, here `UP4`.